src/server/services/status.js

```diff
@@ -11,13 +11,25 @@ export async function getAllStatus() {
 }
 
 /**
+ * Fetch a service by its name.
+ *
+ * @param  {String} id
+ * @return {Promise}
+ */
+export async function getStatusGroupedByName() {
+  return await new Status().query(qb => qb.groupBy('name')
```

Comment:
@romitamgai @kabirbaidhya @mesaugat @pratishshr raw query or query builder, which shall we got with?